### PR TITLE
feat: Add a loader till dashboard app is loaded

### PR DESCRIPTION
**Pull request by fayazara**
## Description
Adds a loader for dashbaord apps, for better experience, as the current screen just shows a blank box.

Screenshot2022-11-08 at 10 53 43

### CODE REVIEW

**Review by nithindavid — APPROVED** [review]
🚀

**Review by iamsivin — APPROVED** [review]
@fayazara LGTM 🎉. Have a small doubt that the loader of the dashboard app is only showing for the first time after refreshing or the app load. Is it okay? 
https://www.loom.com/share/3c8a63fb09e84c06991fe26769e7b926

File: app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue [hunk 1]
```diff
@@ -16,7 +21,11 @@
 </template>
 
 <script>
+import LoadingState from 'dashboard/components/widgets/LoadingState.vue';
```
Comment (iamsivin) [inline app/javascript/dashboard/components/widgets/DashboardApp/Frame.vue:24]:
```suggestion
import LoadingState from 'dashboard/components/widgets/LoadingState';
```

**Comment by pranavrajs** [comment]
@iamsivin Nice catch! @fayazara It should reset the loading state is we are moving to another tab.

**Pull request merged**